addons/console/Console.gd

```diff
@@ -26,6 +26,7 @@ var console_commands := {}
 var console_history := []
 var console_history_index := 0
 
+var pause_enabled: bool = true
```

Comment:
Given the confusion from the original issue, it probably makes sense to have pause disabled by default.